code/modules/cargo/packs.dm

```diff
@@ -36,6 +36,14 @@ datum/supply_pack
 
 /datum/supply_pack/emergency
 	group = "Emergency"
+	
+/datum/supply_pack/emergency/vehicle
+	name = "All-Terrain Vehicle"
+	cost = 2000
```

Comment:
i really wanna keep it cheap because it only has one ATV and greytiders everywhere will foam at the mouth when they see it.